Assets/RedBlueGames/MulliganRenamer/Editor/MulliganRenamerPreviewPanel.cs

```diff
@@ -174,14 +170,42 @@ private static bool DrawPreviewRow(Rect rowRect, PreviewRowModel info, PreviewRo
             thirdColumnRect.height = rowRect.height;
             if (style.ThirdColumnWidth > 0)
             {
-                var size = style.ThirdColumnStyle.CalcSize(new GUIContent(info.FinalName));
-                GUI.DrawTexture(new Rect(thirdColumnRect.x, thirdColumnRect.y, size.x, size.y), tex, ScaleMode.StretchToFill);
+                ApplyBackgroundColorToWhitespaces(thirdColumnRect, style.ThirdColumnStyle, info.FinalName);
                 EditorGUI.LabelField(thirdColumnRect, info.FinalName, style.ThirdColumnStyle);
             }
 
             return isDeleteClicked;
         }
 
+        private static void ApplyBackgroundColorToWhitespaces(Rect rect, GUIStyle style, string content)
+        {
+            if (!content.Contains("<color=#"))
```

Comment:
Dumb edge case we probably don't need to consider but this will break if the file being renamed contains "<color=#". 

Dang kinda wish I used the RenameResult in the PreviewRowModels instead of the raw string, cause you'd have access to the diff. 

But, I say we just ignore this case for now. There's a workaround in case anyone actually encounters this (users can first rename their own color tags with some token then replace them back in.)

Reply:
I think we don't even need to consider this edge case, since `>` and `<` are invalid charactes to have in file names. ([See here](https://en.wikipedia.org/wiki/Filename))